src/Avalonia.Base/PixelSize.cs

```diff
@@ -67,13 +68,56 @@ public PixelSize(int width, int height)
         /// </summary>
         /// <param name="s">The string.</param>
         /// <returns>The <see cref="PixelSize"/>.</returns>
+        /// <exception cref="FormatException"/>
         public static PixelSize Parse(string s)
         {
```

Comment:
Nitpicking here but this overload can simply call `Parse(s, ',')` now that you've added this overload.
Ideally we'd use an optional argument instead of the two overloads, but that would be a breaking change.